startup.sh

```diff
@@ -4,48 +4,67 @@ set -euo pipefail
 
 BUILD_DIR=".local_yaml"
 PKI_DIR=".local_pki"
-COMPONENTS="proxy-node-gateway proxy-node-translator stub-connector"
+COMPONENTS="proxy-node-gateway proxy-node-translator stub-connector softhsm"
 PN_PROJECT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 PKI_OUTPUT_DIR="${PN_PROJECT_DIR}/${PKI_DIR}"
+MINIKUBE_IP="${PKI_OUTPUT_DIR}/minikube_ip"
 
-(minikube status | grep -i running) || minikube start --memory 4096
+(minikube status | grep -i running) || minikube start --memory 4096 "$MINIKUBE_ARGS"
 
 mkdir -p "${BUILD_DIR}"
+
+helm_tpl_args=""
+
 for component in $COMPONENTS; do
 	tag="local-$(tar c $component | md5sum | awk '{print $1}')"
 	image="govukverify/${component}:${tag}"
 	echo "building $component as $image"
 	if (eval $(minikube docker-env --shell bash) && docker inspect --type=image "${image}" >/dev/null 2>&1); then
 		echo "already built"
 	else
-		docker build --file "${component}/Dockerfile" --build-arg "component=${component}" -t "${image}" .
-		docker save "${image}" | (eval $(minikube docker-env --shell bash) && docker load)
+  docker build --file "$component/Dockerfile" --build-arg "component=${component}" -t "${image}" .
```

Comment:
Indentation has gone funny on this file.

Reply:
That's because someone had converted everything to tabs >:o